scripts/combine/setupCombine.py

```diff
@@ -409,107 +411,125 @@ def assertSample(name, startsWith=["W", "Z"], excludeMatch=[]):
-                                    preOpMap={
-                                        m.name: (lambda h: hh.addHists(h[{ax: hist.tag.Slicer()[::hist.sum] for ax in poi_axes}],
-                                                                        h[{"ptVgenSig": hist.tag.Slicer()[0:h.axes["ptVgenSig"].size:hist.sum],
-                                                                            "absYVgenSig": hist.tag.Slicer()[hist.overflow:]}],
-                                                                        scale2=args.scaleNormXsecHistYields)
-                                                    ) if sign in m.name else (lambda h: h[{ax: hist.tag.Slicer()[::hist.sum] for ax in poi_axes}]) for g in cardTool.procGroups["signal_samples"] for m in cardTool.datagroups.groups[g].members
-                                    },
-                                    )
-                # ptV OOA and yV OOA, integrate helicities
-                cardTool.addSystematic("yieldsTheoryAgnostic",
-                                    rename=f"yieldsTheoryAgnostic_OOA_ptVyV_{label}{sign}",
-                                    **noi_args,
-                                    mirror=True,
-                                    scale=0.5,
-                                    processes=["signal_samples"],
-                                    baseName=f"{nuisanceBaseNameOOA}{sign}_",
-                                    noConstraint=True if args.priorNormXsec < 0 else False,
-                                    systAxes=["ptVgenSig", "absYVgenSig","helicitySig"],
-                                    formatWithValue=[None,None,"low"],
-                                    labelsByAxis=["PtVBin", "YVBin", "Helicity"],
-                                    systAxesFlow=["ptVgenSig", "absYVgenSig"], # this can activate nuisances on overflow bins, mainly just ptV and yV since the helicity axis has no overflow bins
-                                    skipEntries=[{"helicitySig" : [6,7,8]}], # removing last three indices out of 9 (0,1,...,7,8) corresponding to A5,6,7
-                                    preOpMap={
-                                        m.name: (lambda h: hh.addHists(h[{ax: hist.tag.Slicer()[::hist.sum] for ax in poi_axes}],
-                                                                        h[{"ptVgenSig": hist.tag.Slicer()[hist.overflow:],
-                                                                            "absYVgenSig": hist.tag.Slicer()[hist.overflow:]}],
-                                                                        scale2=args.scaleNormXsecHistYields)
-                                                    ) if sign in m.name else (lambda h: h[{ax: hist.tag.Slicer()[::hist.sum] for ax in poi_axes}]) for g in cardTool.procGroups["signal_samples"] for m in cardTool.datagroups.groups[g].members
-                                    },
-                                    )
-
+            if args.theoryAgnosticPolVar:
```

Comment:
Can this go into a helper? Either in combine_helpers.py directly or combine_theoryagnostic_helpers.py or something

Reply:
Alright, I personally prefer to see directly what is done, but I can move everything to a helper if people prefer that solution.